modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
 
 ### Fixed
+- Dragging CEF element toward the bottom edge would cause the page to scroll up. #TINY-9025
```

Comment:
Not sure that the audience knows what a CEF is? So maybe:

```suggestion
- Dragging a noneditable element toward the bottom edge would cause the page to scroll up. #TINY-9025
```